src/commons/assessment/AssessmentTypes.ts

```diff
@@ -99,6 +100,7 @@ export type AssessmentConfiguration = {
 
 export interface IProgrammingQuestion extends BaseQuestion {
   answer: string | null;
+  lastModifiedAt: string;
   autogradingResults: AutogradingResult[];
```

Comment:
I think this should be optional instead, so that you can instantiate a new question without it being set.

Reply:
I think this is for the answer. If an answer is created it is technically lastModified at the created time right?

Reply:
Yup so I left it as is for the answer type, but this is the question type